models/custom-application/src/custom-application-menu-link/generator.ts

```diff
@@ -0,0 +1,21 @@
+import { LocalizedField } from '@commercetools-test-data/commons';
+import { fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { TCustomApplicationMenuLink } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const generator = Generator<TCustomApplicationMenuLink>({
+  fields: {
+    id: fake((f) => f.string.uuid()),
+    createdAt: fake(getOlderDate),
+    updatedAt: fake(getNewerDate),
+    labelAllLocales: fake(() => [LocalizedField.random()]),
+    defaultLabel: fake((f) => f.lorem.word()),
+    permissions: [],
+    icon: '<svg><path fill="#000000" /></svg>',
+    submenuLinks: [], // this field is deprecated
```

Comment:
Let's just remove this

Reply:
removed [92e76c3](https://github.com/commercetools/test-data/pull/542/commits/92e76c38c7471417c1220862aa810eda80064cc0)